app/views/users/_profile.html.erb

```diff
@@ -58,7 +58,7 @@
 
     <div class="crayons-field crayons-field--checkbox">
       <%= f.check_box "profile[display_email_on_profile]",
-                      checked: profile.display_email_on_profile,
+                      checked: @user.setting.display_email_on_profile,
```

Comment:
This seems to have been overlooked during the user settings migration. It still worked because there was some delegation in place but now that we remove the profile field we need to read this value from the correct location.